cesql/cesql_tck/like_expression.yaml

```diff
@@ -93,3 +93,26 @@ tests:
     eventOverrides:
       myext: "abc123123%456_dzf"
     result: false
+  
+  - name: With type coercion from int (1)
+    expression: "234 LIKE '23_'"
+    result: true
+  - name: With type coercion from int (2)
+    expression: "2344 LIKE '23%'"
+    result: true
+  - name: With type coercion from int (3)
+    expression: "2344 LIKE '23%'"
+    result: false 
+
+  - name: With type coercion from bool (1)
+    expression: "TRUE LIKE 'tr%'"
```

Comment:
where does the spec say "TRUE" is a bool? I only see "true"

Reply:
Ah I wasn't referring to the spec here, rather the existing tck tests: https://github.com/cloudevents/spec/blob/755e45863e136ee35ac26941405c75666c091c9b/cesql/cesql_tck/casting_functions.yaml#L30-L35

Do you think I should switch this to `true` here?

Reply:
You might be in a better position than me since my only knowledge of the spec is based on this PR and my current reading of the spec to review your PR ;-) , but is the "expression" string meant to adhere to the syntax defined by the spec? If so, then it seems like it may need to be `true` or the spec needs to allow for both upper and lower case bools. Actually, the spec should probably say which things are case sensitive or not, in general.

Reply:
@duglin from https://github.com/cloudevents/spec/blob/main/cesql/spec.md#2-language-syntax it looks like the grammar is case-insensitive so both `true` and `TRUE` should be valid. Maybe we can open a new issue to add clarification to the spec where we mention the boolean literals that this is the case?

Reply:
I think that would be good since it may not be clear that "keywords" don't just mean function names, or operators, but also constants.  And thanks for the pointer - I missed that.